net.krafting.PleasureDVR.yml

```diff
@@ -0,0 +1,36 @@
+app-id: net.krafting.PleasureDVR
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: PleasureDVR.py
+finish-args:
+  -  --share=ipc
+  -  --share=network
+  -  --socket=wayland
+  -  --socket=fallback-x11
+  -  --filesystem=xdg-videos
+  -  --talk-name=org.freedesktop.Notifications
+
+modules:
+  - name: PleasureDVR
+    buildsystem: simple
+    build-commands:
+      - install -D PleasureDVR.py /app/bin/PleasureDVR.py
+      - install -Dm 0644 net.krafting.PleasureDVR.desktop  /app/share/applications/net.krafting.PleasureDVR.desktop
+      - install -Dm 0644 net.krafting.PleasureDVR.metainfo.xml  /app/share/metainfo/net.krafting.PleasureDVR.metainfo.xml
+      - install -Dm 0644 images/tmp_logo256x256.png /app/share/icons/hicolor/256x256/apps/net.krafting.PleasureDVR.png
+    sources:
+      - type: git
+        url: https://gitlab.com/Krafting/pleasure-dvr-gtk
+        tag: beta-1.4
+
+  - name: chaturbate-dvr
+    buildsystem: simple
+    build-commands:
+      - install -D chaturbate-dvr /app/bin/chaturbate-dvr
```

Comment:
so you copy directly a binary? Which architecture is that for. How does this work with multiple architecture?

Reply:
It is a python script, which could work with multiple architecture, but I can only test on x86_64 for now, should I precise it somewhere ?

Reply:
no it is not
```
chaturbate-dvr: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, Go BuildID=M0t-ArIGHyRvlVzqWNVN/dkhfMzRfg0OKryxzWioj/VN4nB19Wo7_heGHMkoyC/kdGjZeCMj4Z-UVjv0rPz, with debug_info, not stripped
```

Reply:
you should build it from source instead

Reply:
Oh sorry, I missread the file, my bad. I'll try to build it from source.